samples/spring-customer-registry-views-quickstart/src/main/java/com/example/Main.java

```diff
@@ -1,11 +1,14 @@
 package com.example;
 
+import kalix.springsdk.impl.KalixServer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
 
 @SpringBootApplication
+@Import(KalixServer.class)
```

Comment:
Spring Boot will scan files under package `com.example` and sub-pacakges. This is the package of the main function. We need to import the KalixServer like that so Spring can find it and start it.

Reply:
So this is what a user will need to do as well in their project? Could we ship the `Application` instead and have zero extras on top of the components themselves, or is that a no-no in Spring users eyes?

Reply:
You mean to ship the Main for the user? I'm not sure if that's a good idea. They may need to tweak their applications by adding extra scan paths and other configurations.

My idea was to offer this as part of the template, with a clear comment that this import must be kept around. Users will most probably start from our template. 

Users not using the template will have to copy a few bits from the templates pom.xml to have it working properly. In that case, I think it's ok for them to copy that extra line.

Reply:
Sounds good, I'm not quite familiar with what Spring people expect, I thought it was maybe bootstrapping everything for them unless they needed some specifics. (Creating a main just to call `SpringApplication.run(Main.class, args)` seems kinda boilerplatey, but I get it if you need to do other bootstrapping)